utils/remote.c

```diff
@@ -645,6 +647,28 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
+{
+    tr_variant* labels = tr_variantDictAddList(args, key, 100);
+    while (arg && *arg)
+    {
+        const char* pch = strchr(arg, ',');
+        char* str;
+        if (pch)
+        {
+            str = tr_strndup(arg, pch - arg);
+            arg = pch + 1;
+        }
+        else
+        {
+            str = tr_strdup(arg);
+            arg += strlen(arg);
```

Comment:
There are two strlen's here.

This might be overkill, but what do you think of

```c
const size_t n = strlen(arg);
str = tr_strndup(arg, n);
arg += n;
```